libnavigation-ui/src/test/java/com/mapbox/navigation/ui/route/MapRouteLineTest.kt

```diff
@@ -1775,6 +1768,264 @@ class MapRouteLineTest {
         assertEquals(recreatedMapRouteLine.getPrimaryRoute(), secondRoute)
     }
 
+    @Test
+    fun getRouteLineTrafficExpressionDataWhenUniqueStreetClassDataExists() {
+        val routeAsJsonJson = loadJsonFixture("route-unique-road-classes.json")
+        val route = DirectionsRoute.fromJson(routeAsJsonJson)
+        val distances = route.legs()!!.mapNotNull { it.annotation()!!.distance() }.flatten()
+        val distancesSum = distances.subList(0, distances.lastIndex).sum()
+        val roadClasses = route.legs()?.asSequence()
+            ?.mapNotNull { it.steps() }
+            ?.flatten()
+            ?.mapNotNull { it.intersections() }
+            ?.flatten()
+            ?.filter {
+                it.geometryIndex() != null && it.mapboxStreetsV8()?.roadClass() != null
+            }
+            ?.map { it.mapboxStreetsV8()!!.roadClass() }
+            ?.toList()
+
+        val result = getRouteLineTrafficExpressionData(route)
+
+        assertEquals(distances.size, result.size)
+        assertEquals(distances.first(), result[1].distanceFromOrigin, 0.0)
+        assertEquals(result[0].roadClass, roadClasses!!.first())
```

Comment:
```suggestion
        assertEquals("service", result[0].roadClass)
```

and remove the local `roadClasses` which is outdated anyway